source/includes/_account.md

```diff
@@ -747,23 +747,106 @@ If successful, returns a status code of 200 and a list of all accounts.
 
 ### Search accounts
 
-Search for an account by a specified search string. The search string is compared to the following attributes: `accountId`, `name`, `email`, `companyName`, and `externalKey`. The operation returns the account record in which the search string matches all or part of any one of the attributes `name`, `email`, `companyName`, `externalKey`. However, the string must match the entire attribute in case of `accountId`. 
+Search for an account by a specified search string. Search operation can be of two types as follows:
+
+**Basic:** 
+
+The search string is compared to the following attributes: `accountId`, `name`, `email`, `companyName`, and `externalKey`. The operation returns the account record in which the search string matches all or part of any one of the attributes `name`, `email`, `companyName`, `externalKey`. However, the string must match the entire attribute in case of `accountId`. 
 
 For example : An account with `name` *exampleaccount* can be searched using *example* as the search string. However, an account with `accountId` *9254283b-be25-45f1-97c3-b902f4d18bab* cannot be searched using only *9254283b* as the search string.
 
+**Advanced**:
+
+Advanced search allows filtering on the specified fields. The prefix marker `_q=1` needs to be specified at the beginning of the search key to indicate this is an advanced query. 
+
+Some advanced search key examples:
+
+* _q=1&email=john@acme.com&currency=USD - Return accounts where `email` is `john@acme.com` and `currency` is `USD`
+* _q=1&address2[like]=Poit%  - Returns accounts where `address2` starts with `Poit`
+* _q=1&address2=Poitier&currency[neq]=MXN - Returns accounts where `currency` is not `MXN`
+* _q=1&billing_cycle_day_local[gte]=31&currency=USD - Returns accounts where `BCD` is greater than `31` and `currency` is `USD`.
+
+The following fields can be specified as part of the search key: 
+* id
+* external_key
+* email
+* name
+* first_name_length
+* currency
+* billing_cycle_day_local,
+* parent_account_id
+* is_payment_delegated_to_parent,
+* payment_method_id
+* reference_time
+* time_zone
+* locale
+* address1
+* address2
+* company_name
+* city
+* state_or_province
+* country
+* postal_code
+* phone
+* notes
+* migrated
+* created_by
+* created_date
+* updated_by
+* updated_date
+
+The following operators can be specified:
+
+* AND
```

Comment:
In the examples, we use lowercase. This might be confusing to put them upper cased here (even though both work).

Also, maybe let's explain that the search query is formed of `<field>[<operator>]=value` elements (where `[<operator>]` is optional)?

Reply:
Fixed.